js/utils/dom.js

```diff
@@ -130,3 +130,11 @@ export function handleLinks(el) {
     e.preventDefault();
   });
 }
+
+/**
+ * This will scroll the pageContainer to the top of the contentFrame. Use this when calling
+ * scrollIntoView on a page view's root element doesn't scroll the pageContainer to zero.
+ */
+export function scrollPageIntoView() {
+  document.getElementById('pageContainer').scrollIntoView();
+}
```

Comment:
I added this function because the normal scroll into view was leaving the search page partially scrolled. This happened because the actual scrolling container is the grandparent of the search element, #contentFrame, and you want to scroll #pageContainer, not the search view's element.